source/dub/dub.d

```diff
@@ -247,6 +247,75 @@ class Dub {
 		m_project = new Project(m_packageManager, pack);
 	}
 
+	/** Loads a single file package.
+
+		Single-file packages are D files that contain a package receipe comment
```

Comment:
The doc comments are currently all done that way (using a single tab to indent the intermediate lines). With a 4-space tab stop they happen to line up, but if we want to aim for *proper* alignment, we'd have to switch to prefixing with "*" to avoid .editorconfig getting in the way.